lib/std/Build/Step/CheckFile.zig

```diff
@@ -2,6 +2,11 @@
 //! TODO: make this more flexible, supporting more kinds of checks.
 //! TODO: generalize the code in std.testing.expectEqualStrings and make this
 //! CheckFileStep produce those helpful diagnostics when there is not a match.
+const CheckFileStep = @This();
+const std = @import("std");
+const Step = std.Build.Step;
+const fs = std.fs;
+const mem = std.mem;
```

Comment:
Saw `const` at bottom or at the top of the file... 
There are more `const` at the top for Steps files so I took advantage of doing this PR to make things more consistent.